translations/messages.fr.xlf

```diff
@@ -0,0 +1,3061 @@
+      </trans-unit>
+      <trans-unit id="4172293183843503071" datatype="html">
+        <source>This move is a ko, you must play somewhere else before you can play in this intersection again.</source>
+        <target>Ce mouvement est un ko, vous devez jouer ailleurs avant de pouvoir rejouer sur cette intersection.</target>
+      </trans-unit>
+      <trans-unit id="4507691580180510561" datatype="html">
+        <source>We are in counting phase, you must mark stones as dead or alive or accept current board.</source>
+        <target>Nous somme dans la phase decomptage, vous devez marquer les pierres comme mortes ou vivantes, ou bien accepter l'état actuel du plateau..</target>
+      </trans-unit>
+      <trans-unit id="4683884757780403263" datatype="html">
+        <source>You cannot accept before the counting phase.</source>
+        <target>Vous ne pouvez pas accepter avant la phase de comptage.</target>
+      </trans-unit>
+      <trans-unit id="7258684846942631624" datatype="html">
+        <source>This intersection is already occupied.</source>
+        <target>Cette intersection est déjà occupée.</target>
+      </trans-unit>
+      <trans-unit id="3878972107071324960" datatype="html">
+        <source>You cannot commit suicide.</source>
+        <target>Vous ne pouvez pas vous suicider.</target>
+      </trans-unit>
+      <trans-unit id="1472088308118018916" datatype="html">
+        <source>Preliminary information</source>
+        <target>Informations préalables</target>
+      </trans-unit>
+      <trans-unit id="352355973280326398" datatype="html">
+        <source>The game of Go is played on a board called a Goban, and the stones are placed on the intersections.
+        The traditional board is made of 19x19 intersections, but on this website we have the 13x13 board.
+        (For shorter parts, 9x9 and 5x5 boards exist, but are not yet available here).
+        For this tutorial, we will use a small 5x5 for pedagogical purposes.</source>
+        <target>Le jeu de Go se joue sur un plateau appelé Goban, et les pierres sont placées sur les intersections.
+        Le plateau traditionnel fait 19x19 intersections, mais le 13x13 est implémenté sur ce site.
+        (Pour des parties plus courtes, le 9x9 et 5x5 existent, mais ne sont pas encore disponibles).
+        Pour ce tutoriel, nous utiliserons un petit 5x5 à fin pédagogique.</target>
+      </trans-unit>
+      <trans-unit id="5188967317336954433" datatype="html">
+        <source>The goal of the game is to have the most points at the end of the game.
+        We call &quot;territories&quot; the intersections that are empty and isolated from the rest of the Goban by the stones of a single player.
+        Here, Dark has 9 territories on the left, and Light has 8 on the right.
+        The top part belongs to no one.
+        The score at the end of the game is the sum of the territories and the captures.</source>
```

Comment:
"Each player's score at the end of the game is the sum of that player's territories and captures."
et
"Le score d'un joueur en fin de partie correspond à la somme de ses territoires et captures."
non ?

Reply:
mieux oui